kernel/aster-nix/src/syscall/mod.rs

```diff
@@ -583,6 +584,7 @@ pub fn syscall_dispatch(
         SYS_EVENTFD2 => syscall_handler!(2, sys_eventfd2, args),
         SYS_EPOLL_CREATE1 => syscall_handler!(1, sys_epoll_create1, args),
         SYS_PIPE2 => syscall_handler!(2, sys_pipe2, args),
+        SYS_DUP3 => syscall_handler!(3, sys_dup3, args),
```

Comment:
Need to update the Linux compatibility page of the Asterinas Book.

Reply:
Get it!